docs/CHANGELOG.asciidoc

```diff
@@ -38,15 +38,484 @@ Review the following information about the {kib} 8.3.0 release.
+* Support term joins for Elasticsearch document source with vector tile scaling {kibana-pull}129771[#129771]
+* Allow feature editing for document layers with "applyGlobalTime", "applyGlobalQuery", and joins {kibana-pull}124803[#124803]
+
+Observability::
+* Bumps synthetics integration package to 0.9.4 {kibana-pull}133423[#133423]
+* Immediately re-run monitors in the synthetics service when they're edited {kibana-pull}132639[#132639]
+* Enables log flyouts on APM logs tables {kibana-pull}132617[#132617]
+* Adds logging to Metric Threshold Rule {kibana-pull}132343[#132343]
+* Adds Page load distribution chart to overview page {kibana-pull}132258[#132258]
+* Show experimental locations only when a particular flag is enabled {kibana-pull}132063[#132063]
+* Trace explorer {kibana-pull}131897[#131897]
+* Static Java agent version list becomes stale quickly {kibana-pull}131759[#131759]
+* Adds recovery context to Log Threshold rule {kibana-pull}131279[#131279]
+* Adds recovery context to the Metric Threshold rule {kibana-pull}131264[#131264]
+* Adds context variables to recovery alerts for Inventory Threshold rule {kibana-pull}131199[#131199]
+* Copy alert state to alert context and implement alert recovery {kibana-pull}128693[#128693]
+* Progressive fetching (experimental) {kibana-pull}127598[#127598]
+* Replace environment dropdown with SuggestionsSelect in landing pages and service overview page {kibana-pull}126679[#126679]
+* Store Logs UI settings in a dedicated `infrastructure-monitoring-log-view` saved object {kibana-pull}125014[#125014]
+
+Platform::
+* The saved objects management table has a new "Last updated" column to easily access recently modified saved objects {kibana-pull}132525[#132525]
+* Sync panels tooltips on dashboard level {kibana-pull}130449[#130449]
+
+Querying & Filtering::
+* Support fields custom label on filter editor {kibana-pull}130533[#130533]
+* Allows comma delimiter on the filters multiple selections dropdowns {kibana-pull}130266[#130266]
+
+Security::
+* Warn & disallow creating role with existing name {kibana-pull}132218[#132218]
+* Improves Sessions tab {kibana-pull}131583[#131583]
+* Adds experimental `csp.disableUnsafeEval` config option to remove the unsafe-eval CSP {kibana-pull}124484[#124484]
+
+[float]
+[[fixes-v8.3.0]]
+=== Bug fixes
+Alerting::
+* Don't load connectors and connector types when there isn't an encryptionKey {kibana-pull}133335[#133335]
+* Adds cloud icon "ess-icon" at the end of the config keys in "alerting" {kibana-pull}131735[#131735]
+* Fixes optional spaceId in rules_client {kibana-pull}130704[#130704]
+
+Dashboard::
+* Adds Fatal Error Handling {kibana-pull}133579[#133579]
+* Hide in Print Mode {kibana-pull}133446[#133446]
+* Send Control State to Reporting Via Locator {kibana-pull}133425[#133425]
+* Fixes new controls causing unsaved changes bug {kibana-pull}132850[#132850]
+* Fixes the listingLimit settings url {kibana-pull}129701[#129701]
```

Comment:
Wondering if this should go under the `dashboard` section? It's a fix for the generic listing page, so it applies to Maps, Visualize, Graph, and Dashboard. Maybe it should go under the management section